torch/jit/_fuser.py

```diff
@@ -48,6 +48,11 @@ def fuser(name):
         torch._C._jit_override_can_fuse_on_gpu(False)
         torch._C._jit_set_texpr_fuser_enabled(False)
         torch._C._jit_set_nvfuser_enabled(True)
+    elif name == 'none':  # Turn Pytorch fuser off
```

Comment:
Suggest separate PR into different topics: JIT side changes and exporter side changes. The benefits are
* Easier to track changes and tests.
* We fully own exporter side, so it can be reviewed and merged much faster, if without JIT changes.
* Don't need to rework on the whole thing, in case of the PR being reverted after merged.